service/src/main/java/org/mskcc/cmo/metadb/service/MetadbSampleService.java

```diff
@@ -15,12 +15,10 @@
             throws Exception;
     List<String> getPooledNormalsBySample(MetadbSample metaDbSample) throws Exception;
     MetadbSample getMetadbSample(UUID metaDbSampleId) throws Exception;
-    MetadbSample getMetadbSampleByRequestAndAlias(String requestId,
-            SampleAlias igoId) throws Exception;
-    MetadbSample getMetadbSampleByRequestAndIgoId(String requestId, String igoId) throws Exception;
-    List<SampleMetadata> getSampleMetadataListByCmoPatientId(String cmoPatientId) throws Exception;
-    List<MetadbSample> getAllSamplesByRequestId(String requestId) throws Exception;
-    List<SampleMetadata> getSampleMetadataHistoryByIgoId(String igoId) throws Exception;
+    MetadbSample getResearchMetadbSampleByRequestAndIgoId(String requestId, String igoId) throws Exception;
```

Comment:
we can remove `Metadb` from this method name

`getResearchMetadbSampleByRequestAndIgoId` --> `getResearchSampleByRequestAndIgoId`